src/deserialize/parse.rs

```diff
@@ -1424,4 +1469,72 @@ mod test {
         assert_eq!(res, Edn::Double(0.00000000000501122771367421.into()));
         assert_eq!(res.to_string(), "0.00000000000501122771367421");
     }
+
+    #[test]
+    fn parse_0x_ints() {
+        let mut edn = "0x2a".chars().enumerate();
+        let res = parse(edn.next(), &mut edn).unwrap();
+        assert_eq!(res, Edn::UInt(42));
+
+        let mut edn = "-0X2A".chars().enumerate();
+        let res = parse(edn.next(), &mut edn).unwrap();
+        assert_eq!(res, Edn::Int(-42));
+    }
+
+    #[test]
+    fn parse_radix_ints() {
+        let mut edn = "16r2a".chars().enumerate();
+        let res = parse(edn.next(), &mut edn).unwrap();
+        assert_eq!(res, Edn::UInt(42));
+
+        let mut edn = "8r63".chars().enumerate();
+        let res = parse(edn.next(), &mut edn).unwrap();
+        assert_eq!(res, Edn::UInt(51));
+
+        let mut edn = "36rabcxyz".chars().enumerate();
+        let res = parse(edn.next(), &mut edn).unwrap();
+        assert_eq!(res, Edn::UInt(623741435));
+
+        let mut edn = "-16r2a".chars().enumerate();
```

Comment:
@evaporei need you opinion here, I don't like the idea of having a negative sign before the radix, like `-16r`, and I would be keen to using something like `-128426_r9`, what do you think? 

I also know it break style with the `0x`and `0b`, but I found this really hard to read. Another hard example would be `"36rabcxyz"`. I would expect this to be a symbol

Reply:
as responded later, this is the expected behavior in clojure